bin/php/Dockerfile

```diff
@@ -17,7 +17,9 @@ RUN yum --releasever=2017.03 install \
     openssl-devel \
     libpng-devel \
     libjpeg-devel \
-    curl-devel -y
+    curl-devel \
+    findutils \
```

Comment:
The `pecl` uses the `find` command, so we need to install it first.

Reply:
👍 could you put that in a comment in the code? (that way the explanation is not lost once this is merged)